### ISSUE TITLE
Keyboard Access Violations in cordova windows 5.0.0

### ISSUE THREAD

**Issue by SwiftyAlex** [apache/cordova-windows#269]
my project is running Windows@5.0.0. Everything works fine on all platforms, apart from on one Windows phone that is running 10.0.15254.248. The other windows phone I have, which is running: 10.0.14393.2068, runs it perfectly fine.

Whenever you click on an input, and then type a key, it instantly crashes the app, with the following exception:

JavaScript runtime error: Unable to get property 'handleEvent' of undefined or null reference
The program '[7608] WWAHost.exe' has exited with code -1073741819 (0xc0000005) 'Access violation'.

A previous version of the app installed on the same phone works fine, however this is an old build and I have no access to this code. I have tried reverting to the last code I have and the issue still persists, and have updated and reverted versions as best I can to try and find a working combination.

Has anyone come across this before? Below is the offending HTML - just a standard input.

```
<label for="x" class="y">Input your z</label>
```

**Comment by janpio** [comment]
Latest `cordova-windows` is 6.0.0 - but that shouldn't have any influence on your problem. I updated your issue title (and made the `label` visible by formatting it as code).

Do you know which version of `cordova-windows` the working app is using?

**Comment by alexthirtyseven** [comment]
They were both using 5.0.0 - I'll try bumping to 6.0.0

**Comment by alexthirtyseven** [comment]
@janpio  The issue still persists when bumping the app to 6.0.

**Comment by janpio** [comment]
As I said, "that shouldn't have any influence on your problem." 

From your description, could it be that building it with a newer SDK version causes the issue for that one special device? I guess the older version of the app, that is working, might have been built with another local setup than what you recreated by going back to the old code?

**Comment by DennisSmolek** [comment]
What listeners are attached to that input? `handleEvent` is a low level JS function that attaches to eventHandlers. Cordova/win uses the UWP JS so I'd take a look at whatever event listeners are firing/listening. 

Another thing, that `<label>` example is missing a few things:
`<label>Click me <input type="text" /></label>` would work, but your example has no input. 

Does the `for="x"` match with your input?: `<input id="x" type="text" />`

**Comment by janpio** [comment]
Ping @alexthirtyseven

**Comment by alexthirtyseven** [comment]
I no longer have access to the project that this was an issue on, but the comment from @DennisSmolek was all tested before I posted this issue.
